modules/entity/src/models.ts

```diff
@@ -38,6 +38,10 @@ export type UpdateNum<T> = {
 
 export type Update<T> = UpdateStr<T> | UpdateNum<T>;
 
+export type Map<T> = {
```

Comment:
Recommend renaming this to something more specific. Maybe `UpdateMap`?

Reply:
LGTM!